### PR TITLE
fix: header origin is managed automatically

### PR DESCRIPTION
**Pull request by m3taphysics**
## What does this PR change?

Prevents ORIGIN header from being set and thus the following error:

`The header Origin is managed automatically, setting it may have no effect or result in unexpected behavior.`

## How to test the changes?

Test happy paths
Test mini-games
Ensure progress is saved (apart from that one bug we know about)

Ensure the bugs mentioned here still are fixed: https://github.com/decentraland/unity-explorer/pull/1388

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: Explorer/Assets/DCL/WebRequests/RequestEnvelope.cs [hunk 1]
```diff
@@ -110,6 +110,9 @@ private void SetHeaders(UnityWebRequest unityWebRequest)
             for (var i = 0; i < count; i++)
             {
                 WebRequestHeader header = info[i];
+
+                // Fix for: The header Origin is managed automatically, setting it may have no effect or result in unexpected behavior.
+                if (header.Name == "Origin") continue;
```
Comment (AlejandroAlvarezMelucciDCL) [inline Explorer/Assets/DCL/WebRequests/RequestEnvelope.cs:115]: Maybe we could consider having "Origin" to be a constant?

**Review by AlejandroAlvarezMelucciDCL — APPROVED** [review]
LGTM, just left a super minor comment

**Comment by AlejandroAlvarezMelucciDCL** [comment]
Looks way better now 💪 good job!

**Review by pravusjif — APPROVED** [review]
LGTM!

**Pull request closed without merging**